lms/templates/instructor/instructor_dashboard_2/certificates.html

```diff
@@ -68,7 +70,9 @@ <h3 class="hd hd-3">${_("Student-Generated Certificates")}</h3>
                 <input type="submit" class="btn-blue" id="enable-certificates-submit" value="${_('Enable Student-Generated Certificates')}"/>
             </form>
         % else:
-            <p>${_("You must successfully generate example certificates before you enable student-generated certificates.")}</p>
+            <p class="under-heading">
+                ${_("You must successfully generate example certificates before you enable student-generated certificates.")}
```

Comment:
Are student-generated certificates still supported/possible?

I find the text on line 74 confusing (since we're disabling example certs in this PR).

Reply:
Student-generated certificates can still be enabled for instructor-paced courses configured to use web/HTML certificates. In that scenario it looks like this:
![image](https://user-images.githubusercontent.com/3229735/127896395-207fe4ed-a4a6-4357-b1db-810c2319b51e.png)

In the case where a course is configured to use PDF certificates, if they hadn't generated example certificates by 2018-ish (before we ripped out PDF cert support), we will be unable to activate Student-generated certificates and the dashboard will look like this:
![image](https://user-images.githubusercontent.com/3229735/127896722-065dc554-5314-461c-9729-b1ea4c80bf6c.png)